### ISSUE TITLE
sync() with nested models is broken

### ISSUE THREAD

**Issue by Qsppl** [hybridsjs/hybrids#256]
### Expected Behavior
I am calling two models to sync and I expect that the sync will not call the `store.get` / `store.set` methods of that model's store.

example1: https://codepen.io/qsppl/pen/oNRLqLz
```TS
import { define, html, store } from "https://esm.sh/hybrids@8.2.23";

const Project = {
    id: true,
    feature: store.ref(() => Feature),
    [store.connect]: async (id) => {
        throw new Error("The Project model must not be loaded from the API!");
    }
};

const Feature = {
    id: true,
    project: store.ref(() => Project),
    [store.connect]: (id) => {
        throw new Error("The Feature model must not be loaded from the API!");
    }
};

store.sync(Feature, {
    id: "0",
    project: {
        id: "321",
        feature: "2"
    }
});
store.sync(Feature, {
    id: "1",
    project: {
        id: "321",
        feature: "3"
    }
});
```

### Observed Behavior
I call two models to sync and the `store.get` method of that model's store is called once.

### Note
If you do not call the second model to synchronize, then `store.get` will not be called:
example2: https://codepen.io/qsppl/pen/abrZYWv?editors=0010
```TS
store.sync(Feature, {
    id: "0",
    project: {
        id: "321",
        feature: "2"
    }
});
// store.sync(Feature, {
//     id: "1",
//     project: {
//         id: "321",
//         feature: "3"
//     }
// });
```

**Comment by Qsppl** [comment]
Thank you. Take your time, for me this is a non-critical bug.

**Comment by smalluban** [comment]
Fixed in `v9.0.0`

**Comment by Qsppl** [comment]
Thank you
